02-intermediate-ruby/modules-as-namespaces.md

```diff
@@ -1,48 +1,115 @@
 # Modules as Namespaces
 
-### Namespacing
+## Learning Goals
+By the end of this lesson, students should be able to
 
-*Namespacing* is a way of bundling related objects together.  In ruby we use Modules to group related Classes and methods together.  We do this to avoid naming collisions.  
+- Read and write code using Ruby _modules_
+- Use modules to group classes and methods into _namespaces_
 
-For example we might create a class called "Shape" in a graphics program, but that name might be used elsewhere in another library.  When you do Shape.new, how is Ruby to know which Shape class to instantiate?  It's a similar idea to putting files into directories.  Files in the same directory cannot have the same name, but files in different directories may.  
+## Modules
 
-In this example below we can create an Array class and put it into a module.  Notice that it does not affect Ruby's default Array class at all
+Modules are a way to group together related ideas. They're like classes that can't be instantiated, making them a good choice if you need to tie together some methods that don't have any associated state.
+
+We've seen a module before: `Math`. It doesn't makes sense to have an instance of `Math`, it's just a collection of related methods and constants.
```

Comment:
I might re-word this section to exclude the use of "just": The designers of Ruby decided to make `Math` a module rather than a class because it is a collection of...